### PR TITLE
Prerender synthetic interactions

### PR DESCRIPTION
**Pull request by diana-macrometa**
Added documentation for viewing synthetic interactions.

@tushar-macrometa - Should I add info for the other tasks (create, update, delete)? A note in the intro said that right now only Macrometa people can perform those tasks, so I did not document them.

### CODE REVIEW

File: docs/photoniq/prerendering/synthetic-interactions/view-synthetic-interactions.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+sidebar_position: 10
+title: View Synthetic Interactions
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+You have several options for viewing Dynamic Prerendering synthetic interactions, either aggregated or per origin.
+
+- **Synthetic Interactions tab** - You can view synthetic interactions for all origins or for one origin on the Synthetic Interactions tab in the web console.
```
Comment (sandro-macrometa) [inline docs/photoniq/prerendering/synthetic-interactions/view-synthetic-interactions.md]: Here we only allow the user to view specific origin interactions. We don't have an "All origins" option.

File: docs/photoniq/prerendering/synthetic-interactions/view-synthetic-interactions.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+- **REST API** - Macrometa provides several API calls that allow you to view metrics for all origins or for one origin. You can also sort and set any time frame that you need.
```
Comment (sandro-macrometa) [inline docs/photoniq/prerendering/synthetic-interactions/view-synthetic-interactions.md]: For the REST API it is the same, you can only manage specific origin interactions.
Here there is no sort or time frame.

File: docs/photoniq/prerendering/synthetic-interactions/view-synthetic-interactions.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+Dynamic Prerendering provides the following information about synthetic interactions. Field names in the web console are different than those in the API responses. Refer to the API endpoint documentation for a full explanation of the API response.
+
+### Scroll
+
+![Prerendering Synthetic Interactions Tab - Scroll](/img/prerendering/synthetic-interactions-scroll.png)
+
+#### Global
+
+- **Status** - Success status.
```
Comment (sandro-macrometa) [inline docs/photoniq/prerendering/synthetic-interactions/view-synthetic-interactions.md]:
```suggestion
- **Status** - Whether the interaction is active.
```

File: docs/photoniq/prerendering/synthetic-interactions/view-synthetic-interactions.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+- **Steps** - How many steps to finalize scrolling.
+- **Delay** - Delay between each scroll step.
+
+#### Specifics
+
+- **Status** - Success status.
```
Comment (sandro-macrometa) [inline docs/photoniq/prerendering/synthetic-interactions/view-synthetic-interactions.md]:
```suggestion
- **Status** - Whether the interaction is active.
```

File: docs/photoniq/prerendering/synthetic-interactions/view-synthetic-interactions.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+- **URL Paths** - Origin paths where the specific interaction is executed or not.
+
+### Click
+
+![Prerendering Synthetic Interactions Tab - Click](/img/prerendering/synthetic-interactions-click.png)
+
+#### Global
+
+- **Status** - Success status.
```
Comment (sandro-macrometa) [inline docs/photoniq/prerendering/synthetic-interactions/view-synthetic-interactions.md]:
```suggestion
- **Status** - Whether the interaction is active.
```

File: docs/photoniq/prerendering/synthetic-interactions/view-synthetic-interactions.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+- **Sequence** - Interaction execution order.
+- **HTML Selector** - HTML selector where the interaction is executed.
+- **Wait After** - Waiting time (in seconds) after the interaction is executed.
+
+#### Specifics
+
+- **Status** - Success status.
```
Comment (sandro-macrometa) [inline docs/photoniq/prerendering/synthetic-interactions/view-synthetic-interactions.md]:
```suggestion
- **Status** - Whether the interaction is active.
```

File: docs/photoniq/prerendering/synthetic-interactions/view-synthetic-interactions.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+- **URL Paths** - Origin paths where the specific interaction is executed or not.
+- **Wait After** - Waiting time (in seconds) after the interaction is executed.
+
+### Hover
+
+![Prerendering Synthetic Interactions Tab - Hover](/img/prerendering/synthetic-interactions-hover.png)
+
+#### Global
+
+- **Status** - Success status.
```
Comment (sandro-macrometa) [inline docs/photoniq/prerendering/synthetic-interactions/view-synthetic-interactions.md]:
```suggestion
- **Status** - Whether the interaction is active.
```

File: docs/photoniq/prerendering/synthetic-interactions/view-synthetic-interactions.md [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+- **Sequence** - Interaction execution order.
+- **HTML Selector** - HTML selector where the interaction is executed.
+- **Wait After** - Waiting time (in seconds) after the interaction is executed.
+
+#### Specifics
+
+- **Status** - Success status.
```
Comment (sandro-macrometa) [inline docs/photoniq/prerendering/synthetic-interactions/view-synthetic-interactions.md]:
```suggestion
- **Status** - Whether the interaction is active.
```

**Review by sandro-macrometa — CHANGES_REQUESTED** [review]

**Comment by sandro-macrometa** [comment]
@diana-macrometa, this is what we have in the API reference. Maybe you can use the same descriptions:

```
Status - Whether the interaction is active.
Steps - Number of scrolls down to be performed.
Delay - Waiting time (in milliseconds) between scrolls.
URL Paths - Origin paths where the specific interaction is evaluated to be executed based on the operation attribute.
HTML Selector - HTML selector where the interaction is executed.
Wait After - Waiting time (in seconds) after the interaction is executed.
Sequence - Interaction execution order.
```

**Comment by diana-macrometa** [comment]
@sandro-macrometa, applied your suggestions. 

Reviewers, please review.

**Review by sandro-macrometa — APPROVED** [review]

**Pull request merged**